src/groups/bmq/bmqimp/bmqimp_brokersession.t.cpp

```diff
@@ -9812,10 +9811,26 @@ static void test54_distributedTrace()
             // Make sure no other events arrive with the given default timeout.
             bsl::string emptyStr(s_allocator_p);
             ASSERT_NE(events.timedPopFront(&emptyStr,
-                                           mwcsys::Time::nowMonotonicClock() +
+                                           bdlt::CurrentTime::now() +
                                                bsls::TimeInterval(0.01)),
                       0);
         }
+
+        
+        static bool waitForWriteCall(const bsl::deque<mwcio::TestChannel::WriteCall> &calls,
+                                     const bsls::TimeInterval& timeout)
```

Comment:
Similar to some other `waitFor` functions at the top of the file, but the scope is local to test54.